frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/table/GitMacheteGraphTable.java

```diff
@@ -76,7 +76,7 @@ public GitMacheteGraphTable(
     // and all `@NonNull` fields are already initialized. `this` is already `@Initialized` (and not just
     // `@UnderInitialization(GitMacheteGraphTableManager.class)`, as would be with a non-final class) at this point.
 
-    GraphCellPainter graphCellPainter = new GraphCellPainter(GraphEdgeColorToJBColorMapper::getColor) {
+    GraphCellPainter graphCellPainter = new GraphCellPainter(GraphItemColorToJBColorMapper::getColor) {
```

Comment:
A bit unrel: given that you need to override `getRowHeight` here... why not make this value a `GraphCellPainter`'s c'tor param instead?

Reply:
`return GitMacheteGraphTable.this.getRowHeight();` calculates row height and the result changes in particular between initialzatoin (empty table?) and table with content :confused: